Polyhedron/demo/Polyhedron/include/CGAL/statistics_helpers.h

```diff
@@ -71,7 +72,7 @@ void compute_angles(Mesh* poly,Tester tester , double& mini, double& maxi, doubl
     typename Traits::Vector_3 bc(b, c);
     double cos_angle = (ba * bc)
       / std::sqrt(ba.squared_length() * bc.squared_length());
-
+    cos_angle = boost::algorithm::clamp<double>(cos_angle, -1.0, 1.0);
```

Comment:
Why use the boost version and not the std::clamp (https://en.cppreference.com/w/cpp/algorithm/clamp)?

Reply:
Because we are not yet at c++17

Reply:
All clear.